base_ical/views/base_ical.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0) -->
+<data>
+    <record id="view_base_ical_form" model="ir.ui.view">
+        <field name="model">base.ical</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <label class="oe_edit_only" for="name" />
+                    <h1><field name="name" /></h1>
+                    <group>
+                    <group name="main">
+                        <field name="model_id" />
+                        <field name="model" invisible="True" />
+                        <field
+                                name="domain"
+                                placeholder="[('user_id', '=', user.id)]"
```

Comment:
Do you want to introduce the domain widget here?
```suggestion
                                placeholder="[('user_id', '=', user.id)]"
                                widget="domain"
                                options="{'model': 'model'}
```

Reply:
I'd love to, but it can't handle expressions - that would need quite some work in the domain widget itself to make this work